bindings/matrix-sdk-crypto-js/src/lib.rs

```diff
@@ -25,10 +25,21 @@ pub mod machine;
 pub mod requests;
 pub mod responses;
 pub mod sync_events;
+mod tracing;
```

Comment:
I was hesitant on this one. I wanted to expose `Tracing` but with `Tracing.install` to return `null` if tracing is disabled. I bet it's better than testing if `Tracing` exists or not (which I have always found weird).

There is not a `#[cfg(feature = "tracing")]` on every line: just on types that are useless if the `tracing` feature is disabled.

Reply:
That's a good idea :-)!

Reply:
Done! The `tracing.rs` file now contains 2 `inner` module: One for `#[cfg(feature = "tracing")]`, and one for `#[cfg(not(feature = "tracing"))]`. It's much simpler, indeed.

The `LoggerLevel` type is common to both implementation.

I've also replaced `Tracing.install` by `new Tracing`, much simpler (since we can install it several times now, cf https://github.com/matrix-org/matrix-rust-sdk/pull/834/commits/c763ce3f4177b71ffa1935569583582599142670).